robot/writer.py

```diff
@@ -17,7 +18,10 @@ def configure_drive():
         drive_hardware.right_slave_falcon.follow(drive_hardware.right_master_falcon)
         drive_hardware.right_master_falcon.setInverted(InvertType.InvertMotorOutput)
 
+    def configure_lighting():
+
```

Comment:
Needs at least one line inside otherwise python will throw an error. If you don't know what to put write something like `return None`.